cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -57,7 +57,7 @@
     protected static final String PASSWORD_VOLUME_MOUNT = "/opt/strimzi/bridge-password/";
 
     // Configuration defaults
-    protected static final int DEFAULT_REPLICAS = 1;
+    protected static final int DEFAULT_REPLICAS = 0;
```

Comment:
TBH, most users would expect it to run. So I think the `DEFAULT_REPLICAS` should stay 1.

Reply:
Are you saying that I should change the test `KafkaBridgeClusterTest.testDefaultValues` instead then?

Reply:
Sorry, I'm not sure I undertsand why. The default number of replicas was 1 before, so the test should work without any changes, or? The scaling to 0 should be used only when user explicitly sets the `replicas` field in the KafkaBridge CR to 0. Not by default when it is not set.

Reply:
` kafkaBridgeCluster.setReplicas`'s condition has changed from `>` to `>=` which breaks the above-mentioned test, thus the question.

Reply:
I guess that might be because the `replicas` is defined as `int` in `KafkaBridgeSpec` class in `api` module which means that if not set it would be `0`. I guess it should probably have the `default` one already there and this whole default part can be probably removed from `KafkaBridgeCluster`?

Reply:
It should be fixed now. Thanks for the clarification!

Reply:
Oh, I didn't. The replica is still 0 in the test. I see there is a `@DefaultValue(1)` annotation in the `getReplicas` method of `KafkaBridgeSpec`, not sure why it still returns 0.

Reply:
I can manually set the instance field to be 1 on initialization do you want me to do that?

Reply:
Right, that is what I meant - sorry for not being clear on that. The `@DefaultValue` is AFAIK just for the docs.

Reply:
Sure, will do it now. Thanks a lot!